### ISSUE TITLE
*.setBackgroundImage seems to be broken for most classes

### ISSUE THREAD

**Issue by salkin-mada** [supercollider/supercollider#6024]
This regards `backgroundImage` / `setBackgroundImage`. And could be related to #2575 and #3703

## Environment

* SuperCollider version: 3.13.0
* Operating system: Arch Linux / MacOs

## Steps to reproduce

Exchange `Slider2D` with `Slider`, `Knob`, `MultiSliderView` or `RangeSlider` 
```supercollider
(
~bounds = Rect(10, 10, 150, 18);
~image.free; ~image = Image.new(SCDoc.helpSourceDir +/+ "images/flowers2.jpg");
~window = Window.new("Image background");
~slider = Slider2D.new(~window, ~bounds).backgroundImage_(~image);
~window.front;
)
```
It seems like View _superclass_ works, but not its _subclasses._  Needs further investigation.

## Expected vs. actual behavior

To be able to set the background image of all View subclasses.
But getting: `ERROR: Primitive '_QObject_InvokeMethod' failed. ...`

**Comment by dyfer** [comment]
Thanks for the report @salkin-mada 
Interestingly, the example that you provided does not trigger an error for me and the slider does get the background image. This is on macOS with the release version of SC 3.13.0, which comes with Qt 5.15.8. 

I'm guessing this might be something about the Qt version/compatibility. Could you confirm which version of Qt SC uses and whether if this is the same version it was compiled against? (See "About Qt" in the IDE's top menu).

**Comment by salkin-mada** [comment]
@dyfer Sorry - I was a bit unclear / fuzzy minded. The example above with `Slider2D` is the exception. Try using `Slider`, `Knob`, `MultiSliderView` or `RangeSlider`.

Here on Arch SCIDE Qt version is 5.15.9.

**Comment by dyfer** [comment]
Yes, I realized that it was about `Slider`, `Knob` etc later and removed my comment. I was indeed able to reproduce the issue. Thanks for submitting!
